config/defaultfilterflags_test.go

```diff
@@ -5,84 +5,98 @@ import (
 
 	"gopkg.in/yaml.v2"
 
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"github.com/zalando/skipper/eskip"
 )
 
-func Test_defaultFiltersFlags_String(t *testing.T) {
+func Test_defaultFiltersFlags_Set(t *testing.T) {
+	oneFilter := eskip.MustParseFilters(`tee("https://www.zalando.de/")`)
+	manyFilters := eskip.MustParseFilters(`ratelimit(5, "10s") -> tee("https://www.zalando.de/")`)
+	manyMoreFilters := eskip.MustParseFilters(`ratelimit(5, "10s") -> tee("https://www.zalando.de/") -> inlineContent("OK\n")`)
+
 	tests := []struct {
-		name    string
-		filters []*eskip.Filter
-		want    string
+		name       string
+		args       []string
+		yaml       string
+		want       []*eskip.Filter
+		wantErr    bool
+		wantString string
 	}{
 		{
-			name:    "test string",
-			filters: []*eskip.Filter{},
-			want:    "<default filters>",
+			name:       "test no filter",
+			args:       nil,
+			want:       nil,
+			wantString: "",
 		},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			dpf := &defaultFiltersFlags{
-				filters: tt.filters,
-			}
-			if got := dpf.String(); got != tt.want {
-				t.Errorf("defaultFiltersFlags.String() = %v, want %v", got, tt.want)
-			}
-		})
-	}
-}
-
-func Test_defaultFiltersFlags_Set(t *testing.T) {
-	oneFilter, _ := eskip.ParseFilters(`tee("https://www.zalando.de/")`)
-	manyFilters, _ := eskip.ParseFilters(`ratelimit(5, "10s") -> tee("https://www.zalando.de/")`)
-	tests := []struct {
-		name    string
-		args    string
-		want    []*eskip.Filter
-		wantErr bool
-	}{
 		{
-			name:    "test no filter",
-			args:    "",
-			want:    nil,
-			wantErr: false,
+			name:       "test empty filter",
+			args:       []string{""},
+			yaml:       "",
+			want:       nil,
+			wantString: "",
 		},
 		{
 			name:    "test error filter",
-			args:    "invalid-filter",
+			args:    []string{"invalid-filter"},
 			wantErr: true,
 		},
 		{
-			name:    "test one filter",
-			args:    `tee("https://www.zalando.de/")`,
-			want:    oneFilter,
-			wantErr: false,
+			name:       "test one filter",
+			args:       []string{`tee("https://www.zalando.de/")`},
+			yaml:       `field: tee("https://www.zalando.de/")`,
+			want:       oneFilter,
+			wantString: `tee("https://www.zalando.de/")`,
 		},
 		{
-			name:    "test many filters",
-			args:    `ratelimit(5, "10s") -> tee("https://www.zalando.de/")`,
-			want:    manyFilters,
-			wantErr: false,
+			name:       "test many filters in one value",
+			args:       []string{`ratelimit(5, "10s") -> tee("https://www.zalando.de/")`},
+			yaml:       `field: ratelimit(5, "10s") -> tee("https://www.zalando.de/")`,
+			want:       manyFilters,
+			wantString: `ratelimit(5, "10s") -> tee("https://www.zalando.de/")`,
+		},
+		{
```

Comment:
Do you think it makes sense to add testcase with invalid yaml, something like:
```
		{
			name: "error during unmarshal",
			args: []string{`ratelimit(5, "10s") -> tee("https://www.zalando.de/")`},
			yaml: `
field:
	- ratelimit(5, "10s") -> tee("https://www.zalando.de/")
	thisStringMakesYamlInvalid
`,
			want:       manyFilters,
			wantString: `ratelimit(5, "10s") -> tee("https://www.zalando.de/")`,
		},
```

Reply:
Added tests for invalid flag and yaml values.